apps/prairielearn/src/tests/testLoadCourse/brokenCourse/infoCourse.json

```diff
@@ -4,5 +4,6 @@
     "title": "Broken course configs",
     "topics": [
         {"name": "basic", "color": "blue3", "description": "Basic topic"}
-    ]
+    ],
+    "tags": ["test1", "test2"]
```

Comment:
Did you mean to commit this change?

Reply:
No, removed in [2d8cf4d](https://github.com/PrairieLearn/PrairieLearn/pull/10973/commits/2d8cf4d59c121453d538338f5912581051a65946).